### PR TITLE
2023 Hyundai Sonata FW - Fingerprint

### PR DESCRIPTION
**Pull request by blantona22**

### CODE REVIEW

**Comment by sshane** [comment]
Can you provide the dongle id that these came from?

**Comment by blantona22** [comment]
10702b68bb2db4f7On Apr 3, 2023, at 4:19 PM, Shane Smiskol ***@***.***> wrote:﻿
Can you provide the dongle id that these came from? It will be in Device settings.

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -539,6 +539,7 @@ class Buttons:
       b'\xf1\x00DN8_ SCC F-CUP      1.00 1.02 99110-L1000         ',
       b'\xf1\x00DN8_ SCC FHCUP      1.00 1.00 99110-L0000         ',
       b'\xf1\x00DN8_ SCC FHCUP      1.00 1.01 99110-L1000         ',
+      b'\xf1\x00DN8_ SCC F-CUP      1.00 1.00 99110-L0000         ',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:542]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -612,9 +615,12 @@ class Buttons:
       b'\xf1\x00HT6WA250BLHT6WA910A1SDN8G25NB1\x00\x00\x00\x00\x00\x00\x96\xa1\xf1\x92',
       b'\xf1\x00HT6WA280BLHT6WAD10A1SDN8G25NB2\x00\x00\x00\x00\x00\x00\x08\xc9O:',
       b'\xf1\x00HT6WA280BLHT6WAD10A1SDN8G25NB4\x00\x00\x00\x00\x00\x00g!l[',
+      b'\xf1\x00HT6WA280BLHT6WAD10A1SDN8G25NB4\x00\x00\x00\x00\x00\x00g!l[',
+      b'\xf1\x87SANDA49495291GD3wx\x87\x88UV\x95i\x88wx\x87\x88\x88\x88\x88\x87w\x88w\x99\xaf\xfc\xffww\x9f\xffHq',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:619]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -612,9 +615,12 @@ class Buttons:
...
       b'\xf1\x00T02601BL  T02730A1  VDN8T25XXX730NS5\xf7_\x92\xf5',
       b'\xf1\x00T02601BL  T02832A1  VDN8T25XXX832NS8G\x0e\xfeE',
       b'\xf1\x00T02601BL  T02900A1  VDN8T25XXX900NSCF\xe4!Y',
+      b'\xf1\x89HT6WAD10A1\xf1\x82SDN8G25NB4\x00\x00\x00\x00\x00\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:623]:
```suggestion
```

**Pull request merged**